samples/subsys/modbus/tcp_server/src/main.c

```diff
@@ -164,7 +202,16 @@ static int init_modbus_server(void)
 		return -ENODEV;
 	}
 
-	return modbus_init_server(server_iface, server_param);
+	err = modbus_init_server(server_iface, server_param);
+
+	if (err < 0) {
+		return err;
+	}
+
+	err = modbus_register_user_fc(server_iface, &modbus_cfg_custom);
+
+	/* ignore if disabled */
+	return (err == -ENOTSUP) ? 0 : err;
```

Comment:
It look like left over from previous patch series, I guess it can be just
`return  modbus_register_user_fc(server_iface, &modbus_cfg_custom);`

Reply:
Ack.